romancal/skycell/tests/skymap_subset.py

```diff
@@ -1,19 +1,17 @@
 from pathlib import Path
 
-import romancal.skycell.skymap as sc
+from romancal.skycell import skymap
 
-LAST_PROJREGION_INDEX = 2
+LAST_PROJREGION_INDEX = 1
 DATA_DIRECTORY = Path(__file__).parent / "data"
 
 if __name__ == "__main__":
-    skymap_subset = sc.SKYMAP.data.copy()
+    skymap_subset = skymap.SKYMAP.model.copy()
 
-    # to maintain the proper indices, the subset must contain all the previous projection regions up to the specified index
-    skymap_subset["roman"]["projection_regions"] = sc.SKYMAP.projection_regions[
+    skymap_subset.projection_regions = skymap.SKYMAP.model.projection_regions[
```

Comment:
I think we should keep this comment:
```suggestion
    # to maintain the proper indices, the subset must contain all the previous projection regions up to the specified index
    skymap_subset.projection_regions = skymap.SKYMAP.model.projection_regions[
```

Reply:
Thanks! it was not my intention to drop it.